resources/cdk/photo_asset_manager/SPECIFICATION.md

```diff
@@ -54,7 +54,7 @@ While developing this example, we have kept two other personas in mind:
 
 # Development / deployment
 
-Refer to [README.md](../tree/main/resources/cdk/photo_asset_manager/README.md) for deployment instructions and [DEVELOPMENT.md](../tree/main/resources/cdk/photo_asset_manager/DEVELOPMENT.md) for instructions on modifying the AWS CDK script.
+Refer to [DEVELOPMENT.md](DEVELOPMENT.md) for instructions on modifying the AWS CDK script and [README.md](README.md) for deployment instructions.
```

Comment:
I think this is switched around. The DEVELOPMENT.md contains per-language deployment instructions and the README.md contains the CDK info.